sound/soc/sof/sof-of-dev.c

```diff
@@ -31,6 +31,35 @@ const struct dev_pm_ops sof_of_pm = {
 };
 EXPORT_SYMBOL(sof_of_pm);
 
+struct snd_sof_of_mach *sof_of_machine_select(struct snd_sof_dev *sdev)
+{
+	struct snd_sof_pdata *sof_pdata = sdev->pdata;
+	const struct sof_dev_desc *desc = sof_pdata->desc;
+	struct snd_sof_of_mach *mach = desc->of_machines;
+
+	if (!mach)
+		return NULL;
+
+	for (; mach->board; mach++) {
+		int score = of_machine_is_compatible(mach->board);
+
+		dev_dbg(sdev->dev, "%s, score:%d, board:%s\n",
+			__func__, score, mach->board);
+		if (score) {
+			sof_pdata->tplg_filename = mach->sof_tplg_filename;
+			if (mach->fw_filename)
+				sof_pdata->fw_filename = mach->fw_filename;
+			dev_info(sdev->dev, "%s ,tplg:%s, fw:%s\n",
```

Comment:
This appears to be a debug message. Should it be removed or rewritten as a sentence?

Reply:
cc @Zames-Chang